DeAceroDLabs/marketplace-frontend
# Feature/dmp 448

## What is done

[[DMP-448] Required message](https://deacero.atlassian.net/browse/DMP-448)

## How is done

- Added validations when a field is required with styles.
- Added validation so that there are not errors to signup.

## How to test

1. Run the project via `npm start`
2. /signup
3. Fill every Field except one.
4. Click on finish.
5. You should see the required message.
6. Try filling form with errors.
7. You should be able to signup.
8. Fill everything correctly.
9. Now you are able to signup.
